src/test/java/com/github/dockerjava/api/command/InspectContainerResponseTest.java

```diff
@@ -51,6 +51,7 @@ public void roundTrip_full() throws IOException {
         assertEquals(response.getVolumesRW()[1].getVolume().getPath(), "/bar/foo/myvol2");
         assertFalse(response.getVolumesRW()[1].getAccessMode().toBoolean());
         assertTrue(response.getVolumesRW()[0].getAccessMode().toBoolean());
+        assertEquals(response.getLogPath(), "/mnt/sda1/var/lib/docker/containers/469e5edd8d5b33e3c905a7ffc97360ec6ee211d6782815fbcd144568045819e1/469e5edd8d5b33e3c905a7ffc97360ec6ee211d6782815fbcd144568045819e1-json.log");
```

Comment:
Bad example was before...
`assertThat(response.getLogPath(), is("/mnt/sda1/var/lib/docker/containers/469e5edd8d5b33e3c905a7ffc97360ec6ee211d6782815fbcd144568045819e1/469e5edd8d5b33e3c905a7ffc97360ec6ee211d6782815fbcd144568045819e1-json.log)"`